### ISSUE TITLE
No example for _.set

### ISSUE THREAD

**Issue by a-roberts** [you-dont-need/You-Dont-Need-Lodash-Underscore#395]
Hi, I run a Code Transformation Guild and, frankly, I love the readme for this repository - it's fantastic. I'm intending to use this as repo as an example as we move developers off Lodash in favour of some of the suggestions here.

Only one problem...we can't see a `._set` example. 

Is this because it's super easy to do, or so general it'd be a pain to do effectively?

I figured I'd be able to see an existing issue for this but...I don't.

Cheers

**Comment by binury** [comment]
Yeah if you look at [what set is doing](https://github.com/lodash/lodash/blob/c7c70a7da5172111b99bb45e45532ed034d7b5b9/src/.internal/baseSet.ts#L17), you'll see it is a bit heavy handed 😥 With all the "safety checks" and edge cases, it's probably ~150 LoC across half a dozen files.


If I wanted an mvp bare set, as a partner to `get` (basically), it would be
```js
function set(obj, path, value) {
  path = Array.isArray(path) ? path.join('.') : path;
  const paths = path.split('.');
  const lastIndex = paths.length - 1;
  
  let nested = obj;
  for (const [i, p] of paths.entries()) {
    if (!nested) { break }
    if (i === lastIndex) {
      nested[p] = value;
    } else {
      nested = nested[p];
    }
  }
  return obj;
}
```
Omitting many checks and the extraneous/niche customizing-path feature, it is much simpler to re-implement like this.
```js

const person = {
  birth: {
    dateTime: '',
    location: {
      state: 'CA',
      city: 'Los Angeles',
      hospital: {
        name: ''
      }
    }
  }
}

set(person, ['birth', 'location', 'hospital', 'name'], 'Hollywood Pediatrics');
set(person, 'vehicle.manufacturer', 'Toyota');

/*
{
  birth: {
    dateTime: '',
    location: { state: 'CA', city: 'Los Angeles', hospital: { name: 'Hollywood Pediatrics' } }
  }
}

*/
```

**Comment by a-roberts** [comment]
@binury - thank you very much, that works perfectly!

Are you thinking of creating a PR to the main readme? No worries if not as folks _could_ find this as a useful reference. Appreciate your help
